src/main/java/com/gmail/nossr50/listeners/EntityListener.java

```diff
@@ -77,43 +109,40 @@ public void onEntityTransform(EntityTransformEvent event) {
 
     @EventHandler(priority = EventPriority.MONITOR, ignoreCancelled = true)
     public void onEntityTargetEntity(EntityTargetLivingEntityEvent event) {
-        if(!ExperienceConfig.getInstance().isEndermanEndermiteFarmingPrevented())
+        if (!ExperienceConfig.getInstance().isEndermanEndermiteFarmingPrevented()) {
             return;
+        }
 
         //It's rare but targets can be null sometimes
-        if(event.getTarget() == null)
-        {
+        if (event.getTarget() == null) {
             return;
         }
 
         //Prevent entities from giving XP if they target endermite
-        if(event.getTarget() instanceof Endermite)
-        {
-            if(event.getEntity() instanceof Enderman) {
-                Enderman enderman = (Enderman) event.getEntity();
+        if (event.getTarget() instanceof Endermite 
+                && event.getEntity() instanceof Enderman) {
+            Enderman enderman = (Enderman) event.getEntity();
```

Comment:
I had them nested for readability, I don't like having too many statements within one if statement